Gemfile

```diff
@@ -1,15 +1,20 @@
 source 'https://rubygems.org'
 
-gem 'sufia', '7.2.0'
-# Hidden Sufia Dependencies
+# semantically-meaningful name for OS-specific gems
+def os_is(re)
+  RbConfig::CONFIG['host_os'] =~ re
+end
+
+gem 'sufia', '~> 7.2.0'
 
+# hidden Sufia dependencies below
 # Sufia pushes flipflop on the app as its not an actual gem, should PR this back somehow?
 gem 'flipflop', git: 'https://github.com/jcoyne/flipflop.git', branch: 'hydra'
 gem 'jbuilder', '~> 2.0' # Hidden Sufia Dependencies...Need PR against sufia...
 # gem 'rsolr', '~> 1.0' # Do we need this? Comment it out for now
 
-# fail update on hydra...was removes in curation concerns...but sufia still needs this
-gem 'blacklight_advanced_search', '~> 6.0'
+# fail on sufia 7.2...they have a lose dependency on CC...which is breaking if it jumps to a higher version
+gem 'curation_concerns', '1.6.1'
```

Comment:
Need to lock this down in our app, as Sufia 7.2 is currently not doing this. Causing a bunch of issues if CC jumps to 1.7.x versions.